Formula/reduce.rb

```diff
@@ -0,0 +1,25 @@
+class Reduce < Formula
+  desc "Tool for adding and correcting hydrogens in PDB files"
+  homepage "https://github.com/rlabduke/reduce"
+  url "https://github.com/rlabduke/reduce/archive/refs/tags/v4.14.tar.gz"
+  version "4.14"
```

Comment:
```suggestion
```
`version` info is not required when the version name can be scanned from the filename.